hack/loadtest/util/placement.go

```diff
@@ -102,21 +101,24 @@ func MeasureOnePlacement(ctx context.Context, hubClient client.Client, deadline,
 			},
 		},
 	}
+
+	applyCRP(crp, crpFile, nsName, clusterNames, fleetSize)
 	defer hubClient.Delete(context.Background(), crp) //nolint
 	if err := hubClient.Create(ctx, crp); err != nil {
 		klog.ErrorS(err, "failed to apply crp", "namespace", nsName, "crp", crpName)
 		return err
 	}
+	crpCount.Inc()
 
 	klog.Infof("verify that the cluster resource placement `%s` is applied", crpName)
 	collectApplyMetrics(ctx, hubClient, deadline, interval, crpName, currency, fleetSize)
 
-	klog.Infof("remove the namespaced resources referred by the placement `%s`", crpName)
-	if err := deleteTestManifests(ctx, hubClient, nsName); err != nil {
-		klog.ErrorS(err, "failed to delete test manifests", "namespace", nsName, "crp", crpName)
-		return err
-	}
-	collectDeleteMetrics(ctx, hubClient, deadline, interval, crpName, clusterNames, currency, fleetSize)
+	//klog.Infof("remove the namespaced resources referred by the placement `%s`", crpName)
```

Comment:
As discussed offline, we can wait for all the CRP conditions to be either True or False before we delete the CRP, that way we can ensure that the CRPs complete before we delete them. And dashboard will show incomplete CRPs but since we wait the incomplete count should go down and complete count should go up before we clean up the CRPs